pkg/ui/workspaces/cluster-ui/src/transactionDetails/transactionDetails.tsx

```diff
@@ -426,6 +452,30 @@ export class TransactionDetails extends React.Component<
               unavailableTooltip
             );
 
+            const isCockroachCloud = useContext(CockroachCloudContext);
+            const insightsColumns = useMemo(
+              () =>
+                makeInsightsColumns(
+                  isCockroachCloud,
+                  this.props.hasAdminRole,
+                  false,
+                ),
+              [isCockroachCloud],
+            );
+            const tableData: InsightRecommendation[] = [];
+            if (transactionInsights) {
+              const tableDataTypes: InsightType[] = [];
```

Comment:
nit: if you're using this to track the insight types seen already, a `set` is prob preferred (although the array size will be small so it's probably not a big deal 🤷 )